src/index.js

```diff
@@ -143,16 +143,24 @@ export const getTargets = (targets = {}) => {
   return targetOps;
 };
 
+export const validateBoolOption = (name, value, defaultValue) => {
+  if (typeof value === "undefined") {
```

Comment:
In the end it's a build tool so we should use when we would like to - hard for us just to implement everything when it's not needed but yeah something I've been struggling with sometimes. Sure we want to keep low dependencies and everyone complaining about Babel install times but hey we aren't getting any help and it just makes developing that much harder